api/src/main/webapp/WEB-INF/queue.yaml

```diff
@@ -4,8 +4,8 @@ queue:
   target: api
 
   # rate parameters
-  bucket_size: 500
```

Comment:
this means something like "execution slots".  The first 500 requests (when the queue has been idle) are executed immediately.

Changing to 1 means that we can only run one batch before needing to free a new execution slot.